releasenotes/notes/1.4/prepare-1.4.0-aee22d10333374c8.yaml

```diff
@@ -0,0 +1,208 @@
+  - |
+    The following uses of the :class:`.BackendProperties` object in the transpilation pipeline
+    have been deprecated as of Qiskit v1.4.0 and will be removed in Qiskit v2.0.0:
+
+    * ``backend_prop`` input argument in :class:`.DenseLayout`
+    * ``properties`` input argument in :class:`.VF2Layout`
+    * ``properties`` and ``coupling_map`` input arguments in :class:`.VF2PostLayout`. Note that ``coupling_map`` was only used in the presence of ``properties``.
+    * ``backend_props`` input argument in :class:`.UnitarySynthesis`
+    * ``backend_properties`` input argument in :class:`.PassManagerConfig`
+    * ``backend_properties`` in  :meth:`.Target.from_configuration`
+    * ``backend_properties`` in :func:`.generate_routing_passmanager`
+    * ``backend_properties`` in :func:`.generate_translation_passmanager`
+
+    The :class:`.BackendProperties` class has been deprecated since Qiskit v1.2.0, together with 
+    other elements from the :class:`.BackendV1` workflow, and will be removed in Qiskit v2.0.0. 
+    The alternative path for communicating hardware information to the transpilation argument is the
+    :class:`.Target` class, which can be set using the ``target`` input argument.
+    Specific instruction properties such as gate errors or durations can be added to a :class:`.Target` upon 
+    construction through the :meth:`.Target.add_instruction` method.
+
+    In the case of :func:`.generate_routing_passmanager` and :func:`.generate_translation_passmanager`,
+    the ``backend_properties`` argument is optional and is superseded when the required ``target`` 
+    argument is populated. Usage of the argument can safely be removed in 1.x as long as you were 
+    passing in a target, which was the recommended use.
+
+  - |
+    The Pulse deprecation in Qiskit v1.3.0, included calibration builder passes
+    such as :class:`.RXCalibrationBuilder`. The
+    :class:`.NormalizeRXAngle` pass is a requirement of :class:`.RXCalibrationBuilder`;
+    hence, it is being deprecated in Qiskit v1.4.0. The :func:`.rzx_templates`
+    function in the calibration module is also being deprecated as it is not used in our codebase.
+
+deprecations_visualization:
+  - |
+    The parameters ``show_idle`` and ``show_barrier`` in the timeline drawers are 
+    deprecated as of Qiskit v1.4.0. The alternatives are, respectively, the 
+    ``idle_wires`` and ``plot_barriers`` parameters, introduced in Qiskit v1.1.0,
+    which are fully equivalent.
+    The legacy parameter names will be removed in Qiskit v2.0.0.
+  - |
+    In Qiskit v1.4.0, the timeline drawer :func:`~qiskit.visualization.timeline.interface.draw` function
+    will issue a deprecation warning if a ``target`` is not specified to get the duration
+    of instructions. From Qiskit v2.0.0 on, ``target`` will be required and 
+    :func:`~qiskit.visualization.timeline.interface.draw` will fail if it is not specified.
+
+upgrade_transpiler:
```

Comment:
Is this an upgrade note? It feels like a deprecation category might be more appropriate or just other note.

Reply:
I think it's a bit in the middle. Other seems fine.